### PR TITLE
Release v0.1.0

### PR DESCRIPTION
**Pull request by szabo137**
This PR merges the latest state of dev into the main and starts the initial release pipeline for  `v0.1.0`.
For more details on this release pipeline, see https://github.com/QEDjl-project/QEDprocesses.jl/issues/32 (cf. https://github.com/QEDjl-project/QEDbase.jl/issues/39)

A full list of PRs that go into this release can be found here: https://github.com/QEDjl-project/QEDprocesses.jl/milestone/2?closed=1

# Todos

- [x] tick up the version in Project.toml
- [x] add CHANGELOG.md with a section to summarize the changes made
- [x] tag the PRs related to this release (using milestones)

### CODE REVIEW

**Comment by szabo137** [comment]
@steindev Is this format for the CHANGELOG the one you assumed?

**Review by tjungni — DISMISSED** [review]

**Review by SimeonEhrig — DISMISSED** [review]

File: CHANGELOG.md
Comment (steindev) [inline CHANGELOG.md:1]: :+1:

**Review by steindev — APPROVED** [review]
Looks good to me (LGTM)

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Changelog
+
+## Version 0.1.0
+
+This is the initial verison of QEDprocesses.
+
+[diff since latest main](https://github.com/QEDjl-project/QEDprocesses.jl/compare/302274695d82225f4a810c252d6919839bc59fd7...release-v0.1.0) [Full list of PRs](https://github.com/QEDjl-project/QEDprocesses.jl/milestone/2?closed=1)
```
[the remark this answers is not in the document]
Comment (tjungni) [reply CHANGELOG.md]: Yes I thought so too. But we can do it starting with the next release

File: .formatting/format_all.jl [hunk 2]
```diff
@@ -0,0 +1,12 @@
+using JuliaFormatter
+
+# we asume the format_all.jl script is located in QEDfields.jl/.formatting
```
[the remark this answers is not in the document]
Comment (szabo137) [reply .formatting/format_all.jl:3]: I opened an issue on that, see #33

**Pull request merged**